bentoml/_internal/configuration/__init__.py

```diff
@@ -42,7 +42,7 @@ def expand_env_var(env_var):
 
 
 def is_pip_installed_bentoml():
-    is_installed_package = hasattr(version_mod, "version_json")
+    is_installed_package = len(version_mod.version_tuple) == 3
     is_tagged = not BENTOML_VERSION.startswith("0+untagged")
```

Comment:
we will also need to change this to `not BENTOML_VERSION.startswith("0.1.")` - looks like setuptools_scm will generate versions like `0.1.dev1392+gddbebc4` when there is no tag

Reply:
done